docs/use_cases/model_applications/medium_range/README.rst

```diff
@@ -1,2 +1,3 @@
 Medium Range
 ------------
+Lower resolution model configuration (>4km) usually producing forecasts out to 7-14days. Also referred to as global models.
```

Comment:
Lower resolution model configuration (>4km) usually producing forecasts out to 7-14 days (also referred to as global models).